### DISCUSSION TITLE
Override default colors in semantic tokens

### DISCUSSION THREAD

**Discussion by jpenna** [chakra-ui/chakra-ui#9192] [General]
### Description

I want to override some colors with my own values and use them in `theme.semanticTokens` but it still uses the default ones.

### Link to Reproduction

Couldn't create because it is set to v3 and I'm using v2

### Steps to reproduce

1. Write custom colors overriding default color names (eg, `blue: { 300: '#000000' }`)
2. Use those colors in semantic tokens (eg, `myColor: blue.300`)
3. Use the semantic token in a component (eg, `backgroundColor: myColor`)
4. See it is using the default value (ie, instead of black, it is still blue)

### Chakra UI Version

2.9.3

### Browser

Edge 130.0.2849.80 

### Operating System

- [ ] macOS
- [ ] Windows
- [X] Linux

### Additional Information

```js
import { extendTheme } from '@chakra-ui/react';

const theme = extendTheme({
  colors: {
    green: {
      // These colors are blue
      50: '#e3f4ff',
      100: '#bedaf3',
      200: '#98c1e6',
      300: '#71a8dc',
      400: '#4b8fd0',
      500: '#3275b7',
      600: '#255b8f',
      700: '#194167',
      800: '#0b2740',
      900: '#000e1b',
    },
  },
  semanticTokens: {
    colors: {
      something: {
        default: 'green.500',
        _dark: 'green.300',
      },
    },
  },
});

export default theme;

```

**Comment by segunadebayo** [comment]
Sorry, there are no more bug fixes for v2.

I'll move this to a discussion to see if others can chime in

**Reply by jpenna** [reply]
That's unfortunate...

Do you know if this was fixed in v3?

**Comment by segunadebayo** [comment]
v2 will not get this fix. v2 is maintenance-only.

In v3, override `theme.tokens.colors.green`, then reference `{colors.green.500}` in semantic tokens.
